src/tools/clippy/clippy_lints/src/redundant_clone.rs

```diff
@@ -113,6 +113,7 @@ impl<'tcx> LateLintPass<'tcx> for RedundantClone {
             }
 
             if let ty::Adt(def, _) = arg_ty.kind() {
+                // FIXME: This is not correct with `#[manually_drop]`, as that is just like any other type.
```

Comment:
If this should get merged like this, could you open up a Clippy issue describing the possible extension that could be done here, please?